src/arch/src/aarch64/gic/regs/dist_regs.rs

```diff
@@ -147,15 +147,13 @@ fn access_dist_reg_list(fd: &DeviceFd, action: &mut Action) -> Result<()> {
     Ok(())
 }
 
-/// Get distributor registers of the GIC.
 pub fn get_dist_regs(fd: &DeviceFd) -> Result<Vec<u32>> {
```

Comment:
We need the function to be `pub`, otherwise it won't be visible outside the file. But we don't re-export it outside of the `regs` module. 

The crate has `#![deny(missing_docs)]`.